### ISSUE TITLE
Filter results based on string matching in report

### ISSUE THREAD

**Issue by sunta3iouxos** [stephenturner/biorecap#4]
is it possible to narrow summaries of pre-prints by using specific strings that are not associated with the subjects so that we could limit those results even more?
for example something like:
```R
biorecap_report(output_dir=".", 
subject=c("bioinformatics"), strings = c("chip-seq"),
model="llama3.1")
```

**Comment by stephenturner** [comment]
Good idea. It's not currently possible with the report function, but you could filter the title and abstract for the presence of that string before further processing. I'll keep this open, potentially add to a future version.
